Source/WebCore/animation/WebAnimation.cpp

```diff
@@ -1482,6 +1482,9 @@ ExceptionOr<void> WebAnimation::commitStyles()
     auto properties = effect->animatedProperties();
     // 2.5 For each property, property, in targeted properties:
     for (auto property : properties) {
+        // FIXME: handle custom properties.
```

Comment:
Reference a bug for this?

Reply:
Was too lazy to add one because I was about to fix this right after this lands. But I’ll add a mention to https://bugs.webkit.org/show_bug.cgi?id=242007 before landing.